### PR TITLE
>shiptest >everyone has ships >no trek uniforms in loadout

### PR DESCRIPTION
**Pull request by Iamgoofball**
I've corrected this obvious oversight.
## About The Pull Request

Fixes the lack of Trek uniforms in the loadout.

## Why It's Good For The Game
picard-meme-facepalm

## Changelog
:cl:
fix: Fixes the lack of Trek uniforms in the loadout.
/:cl:

### CODE REVIEW

File: code/modules/client/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -245,3 +245,35 @@
...
+/datum/gear/uniform/trek_tng_medsci
+	display_name = "next generation medical/science uniform"
+	path = /obj/item/clothing/under/trek/medsci/next
+
+/datum/gear/uniform/trek_ent_command
+	display_name = "enterprise command uniform"
+	path = /obj/item/clothing/under/trek/command/ent
+
+/datum/gear/uniform/trek_ent_engsec
+	display_name = "enterprise engineering/security unifo
```
Comment (triplezeta) [inline code/modules/client/loadout/loadout_uniform.dm]: unifo

Comment (Iamgoofball) [reply code/modules/client/loadout/loadout_uniform.dm]: thanks

**Comment by Apogee-dev** [comment]
loadout menu is already bloated to hell and rank uniforms in particular should probably be left as ship-specific items. no ship currently uses the trek uniforms obviously but it'll be weird to see a bunch of people across multiple factions suddenly start dressing up like solgov's equivalent of weird classical history nerds

**Comment by tiramisuapimancer** [comment]
Frankly if you ask me the loadout menu is practically barren. We could probably do with having subtabs like Citadel does, though.

**Comment by Iamgoofball** [comment]
@Apogee-dev i think you're wrong and there's actually very little choice in the loadout menu right now

File: code/modules/client/loadout/loadout_uniform.dm [hunk 2]
```diff
@@ -245,3 +245,40 @@
 /datum/gear/uniform/maid
 	display_name = "maid dress"
 	path = 	/obj/item/clothing/under/costume/maid
+
+//>shiptest >no trek uniforms in loadout
```
Comment (Fikou) [inline code/modules/client/loadout/loadout_uniform.dm]: comments should describe the current state of code, dont need this

**Review by Fikou — COMMENTED** [review]
i think we have some basic system for locking outfits to jobs, could you do that? command for command roles, engsec for engi sec roles etc etc

**Comment by Iamgoofball** [comment]
i thought all the jobs were unused given the fact its not ss13? if not, give me a list of jobs used then

**Comment by ToasterBan** [comment]
> i thought all the jobs were unused given the fact its not ss13? if not, give me a list of jobs used then

shiptest's custom jobs are based off the different jobs outfits just with different names 

example: the waiter job of the boyardee is just an assistant with different clothing and name

**Comment by Apogee-dev** [comment]
> i think we have some basic system for locking outfits to jobs, could you do that? command for command roles, engsec for engi sec roles etc etc

job-specific loadout items have been broken on shiptest for a pretty long time now

**Comment by Vexylius** [comment]
While actually the entire character creation thingy needs a serious rework, this PR is fine.
Please move the conversation not about this specific PR to #gen-dev please.

**Comment by Fikou** [comment]
we can try this. should probably fix the outfit system in the future (port rohesie's job refactor)

**Pull request merged**